### PR TITLE
[pqdb-frontend] Framework for adding different views + Readme

### PR DESCRIPTION
**Pull request by benj-zen**

### CODE REVIEW

File: pqdb-frontend/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+### Requirements
+
+* Working schema validation script (For install instructions see [here](/README.md) under Schema Validation)
+* Yarn (Installs with node.js, if you use the snap package. Otherwise see [here](https://classic.yarnpkg.com/en/docs/install/))
+
+Before you can run the frontend locally you need to generate the SQLite database and the database diagram. This can be done by running the following command from the root directory of pqdb.
+```
+node tools/validation/validate.js pqdb-frontend/public/pqdb.sqlite pqdb-frontend/src/tables.svg
+```
+If you want to update the data source for the frontend, you need to remove the two files and re-run the command.
```
Comment (taudor) [inline pqdb-frontend/README.md:17]: Maybe add the file names to avoid confusion.

File: pqdb-frontend/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+### Installation
+If not done already, switch to the `tools/pqdb-frontend` directory, then install the dependencies.
```
Comment (taudor) [inline pqdb-frontend/README.md:20]: Folder is `/pqdb-frontend` and not `tools/pqdb-frontend`

File: pqdb-frontend/README.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+```
+
+
+### Run
+
+```
+yarn start
+```
+
+This starts the React app and loads the url in the browser.
```
Comment (taudor) [inline pqdb-frontend/README.md:33]: A little SQL example would be convenient. Especially because the table overview is a little overwhelming at first

Comment (benj-zen) [reply pqdb-frontend/README.md:33]: This is the framework for the frontend views, so SQL examples will come soon, when views are created.

**Comment by dingens** [comment]
Until I've found out how to fix this https://github.com/JoshuaWise/better-sqlite3/issues/403 I can just do static code review :)

I'd suggest `mv pqdb-frontend frontend`. That's it ;)

**Pull request merged**